### PR TITLE
fixes group assignment for sandbox users

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/test/resources/integration-test-data.sql [hunk 1]
```diff
@@ -159,8 +159,12 @@ insert into exam (id, type_id, grade_id, grade_code, student_id, school_id, oppo
   -- groups
 insert into student_group (id, name, school_id, school_year, subject_id, update_import_id, updated, migrate_id) values
   (-10, 'school1 g1', -10, 1997, 1, -1, '1997-07-18 20:14:34.000000', -1),
-  (-20, 'school2 g2', -10, 1997, 2, -1, '1997-07-18 20:14:34.000000', -1);
+  (-20, 'school2 g2', -10, 1997, 2, -1, '1997-07-18 20:14:34.000000', -1),
+  (-30, 'SandboxSchool (subject1,grade1)', -10, 1997, 2, -1, '1997-07-18 20:14:34.000000', -1),
+  (-40, 'SandboxSchool (subject2,grade1)', -10, 1997, 2, -1, '1997-07-18 20:14:34.000000', -1);
```
Comment (esotrope) [inline webapp/src/test/resources/integration-test-data.sql]: @malaffoon this is the correct pattern right? no space after the comma?

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/security/sandbox/JdbcSandboxUserGroupRepository.java [hunk 2]
```diff
@@ -23,20 +23,31 @@
...
                     new MapSqlParameterSource()
                             .addValue("user_login", username)
-                            .addValue("group_name", groupName)
+                            .addValue("organization_name", organizationName)
+                            .addValue("grade_code", gradeCode)
+            );
+        } catch (final Exception exception) {
+            logger.error(
+                    String.format(
+                            "Problems creating user student group for username \"%s\" organization \"%s\" and grade \"%s\"",
```
Comment (malaffoon) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/security/sandbox/JdbcSandboxUserGroupRepository.java]: nit: message is a bit misleading isn't it? i liked the "Problems adding ..." from before

File: webapp/src/test/resources/integration-test-data.sql (hunk 1 above)
Comment (ftdholbrook) [reply webapp/src/test/resources/integration-test-data.sql]: SELECT * FROM warehouse_ca_s001.student_group;
'1','Demo Elementary (ELA,03)','1','2019','2','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'2','Demo Elementary (Math,04)','1','2019','1','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'3','Demo Elementary (ELA,04)','1','2019','2','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'4','Demo Elementary (Math,05)','1','2019','1','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'5','Demo Middle (Math,06)','2','2019','1','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'6','Demo Middle (ELA,06)','2','2019','2','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'7','Demo Middle (Math,07)','2','2019','1','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'8','Demo Middle (ELA,07)','2','2019','2','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'9','Demo Middle (Math,08)','2','2019','1','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'10','Demo Middle (ELA,08)','2','2019','2','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'11','Demo High School (Math,11)','3','2019','1','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'
'12','Demo High School (ELA,11)','3','2019','2','1','dwtest@example.com','9705','9705','0','2019-06-16 22:22:46.968138','2019-06-16 22:22:46.968138'

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/security/sandbox/JdbcSandboxUserGroupRepository.java (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/java/org/opentestsystem/rdw/reporting/security/sandbox/JdbcSandboxUserGroupRepository.java]: 👍

File: webapp/src/test/resources/integration-test-data.sql (hunk 1 above)
Comment (esotrope) [reply webapp/src/test/resources/integration-test-data.sql]: thank you @ftdholbrook

Comment (malaffoon) [reply webapp/src/test/resources/integration-test-data.sql]: that is correct, yes

**Pull request merged**